Cargo.toml

```diff
@@ -185,7 +185,7 @@ tikv_alloc = { path = "components/tikv_alloc" }
 tikv_util = { path = "components/tikv_util", default-features = false }
 collections = { path = "components/collections" }
 time = "0.1"
-tipb = { git = "https://github.com/pingcap/tipb.git", default-features = false }
+tipb = { git = "https://github.com/Xuanwo/tipb.git", branch = "patch-1", default-features = false }
```

Comment:
You can add a single patch in the top-level Cargo.toml for this WIP change.

```
[patch.'https://github.com/pingcap/tipb']
tipb = { branch = "patch-1", git = "https://github.com/Xuanwo/tipb.git", default-features = false }
```

Reply:
Thanks!